gen-kubectldocs/go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/spf13/cobra v1.6.0
 	github.com/spf13/pflag v1.0.5
 	gopkg.in/yaml.v2 v2.4.0
+	k8s.io/apimachinery v0.26.0
 	k8s.io/kubectl v0.26.0
```

Comment:
It should be safe to bump this to v0.28.0 now.

Reply:
My plan was to bump dependencies in a dedicated PR.

Reply:
Okay. Let's get this in first.